src/org/ods/component/Context.groovy

```diff
@@ -324,7 +327,8 @@ class Context implements IContext {
     config.nexusHost.replace("://", "://${config.nexusUsername}:${config.nexusPassword}@")
   }
 
-  String getBranchToEnvironmentMapping() {
+  @NonCPS
+  Map getBranchToEnvironmentMapping() {
```

Comment:
Perhaps have Map<String, String> here?

Reply:
Why not, addressed in d5e030652ad774dc361729a2ca78fc87a200e27e.